### PR TITLE
Bug 1506895 Changed realm in azure blob config

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1506895

cc @tanaka-takayoshi @wzheng1

### CODE REVIEW

**Comment by bfallonf** [comment]
Also for: https://bugzilla.redhat.com/show_bug.cgi?id=1506880

Will put that to QA as well.

**Comment by wzheng1** [comment]
@dongboyan77 Could you please help to review this doc? Thanks!

File: install_config/storage_examples/azure_blob_docker_registry_example.adoc [hunk 1]
```diff
@@ -45,7 +45,7 @@ To create a new registry pod and replace the old pod automatically:
       container: azureblobname
```
Comment (dongboyan77) [inline install_config/storage_examples/azure_blob_docker_registry_example.adoc]: should add `realm: core.windows.net`  in this field, not like below

Comment (bfallonf) [reply install_config/storage_examples/azure_blob_docker_registry_example.adoc]: @dongboyan77 Sorry, I'm not sure if I understand. I've done that on line 48 (below). Should I be placing this somewhere else as well? Should I place it underneath the `container` line above?

Comment (dongboyan77) [reply install_config/storage_examples/azure_blob_docker_registry_example.adoc]: yes, please place it underneath the `container` line above, do not change anything on line 48

Comment (bfallonf) [reply install_config/storage_examples/azure_blob_docker_registry_example.adoc]: Thanks @dongboyan77 . I've changed as suggested. I'll update the BZ. Please let me know if there's anything else for this PR.

**Comment by bfallonf** [comment]
Thanks @dongboyan77 

@openshift/team-documentation Peer review please?

File: install_config/persistent_storage/persistent_storage_azure.adoc [hunk 2]
```diff
@@ -59,8 +59,23 @@ xref:../../install_config/configuring_azure.adoc#install-config-configuring-azur
 for Azure Disk], all that is required for {product-title} and Azure is an Azure
 Disk Name and Disk URI and the `PersistentVolume` API.
 
-[[azure-creating-persistent-volume]]
+[[azure-disk-regional-cloud]]
+== Configuring Azure Disk for regional cloud
+
+Azure has multiple regions on which to deploy an instance. To specify a desired region, add the following to the *_azure.conf_* file:
```
Comment (gaurav-nelson) [inline install_config/persistent_storage/persistent_storage_azure.adoc]: s/Azure/Microsoft Azure 
I noticed this in our Glossary yesterday. Up to you if you want to modify other instances as well.

**Comment by gaurav-nelson** [comment]
@bfallonf just one comment, rest is all good. 🚘

**Comment by bfallonf** [comment]
Thanks @gaurav-nelson . I've updated the three Azure storage sections to "Microsoft Azure" on first mention, then just Azure mentions after. This seems to allign with the Azure docs.

I'll merge.

**Pull request merged**